### PR TITLE
Customize Mixpanel configs

### PR DESCRIPTION
**Pull request by thundron**
Change-type: minor
Signed-off-by: Lorenzo Alberto Maria Ambrosi <lorenzoa@balena.io>

### CODE REVIEW

File: lib/gui/app/modules/analytics.js [hunk 1]
```diff
@@ -17,24 +17,57 @@
 'use strict'
 
 const _ = require('lodash')
-const resinCorvus = require('resin-corvus/browser')
+const resinCorvus = require('/Users/thunder/Documents/Development/resin-corvus/browser')
```
Comment (zvin) [inline lib/gui/app/modules/analytics.js]: this seems wrong

File: lib/gui/app/modules/analytics.js [hunk 2]
```diff
@@ -21,20 +21,53 @@ const resinCorvus = require('resin-corvus/browser')
...
-  }
-})
+const Bluebird = require('bluebird')
+const request = Bluebird.promisifyAll(require('request'))
+const sentry = settings.get('analyticsSentryToken') ||
+  _.get(packageJSON, [ 'analytics', 'sentry', 'token' ])
+const mixpanel = settings.get('analyticsMixpanelToken') ||
+  _.get(packageJSON, [ 'analytics', 'mixpanel', 'token' ])
+
+const configUrl = settings.get('configUrl') || 'http://balena.io/etcher/static'
```
[the remark this answers is not in the document]
Comment (thundron) [reply lib/gui/app/modules/analytics.js]: That's the base folder, the file is this one https://www.balena.io/etcher/static/config.json , probably the name isn't the best though so I'll rename it to something more appropriate

File: lib/gui/app/modules/analytics.js [hunk 3]
```diff
@@ -36,6 +47,38 @@ resinCorvus.install({
...
+      return resinCorvus.setConfigs({
+        mixpanelConfig: (eligible) ? mixpanelConfig : null
+      })
+    }
+
+    throw new Error('Analytics config.json not available')
+  })
+  .catch((err) => {
+    exports.logException(err)
+  })
```
Comment (zvin) [inline lib/gui/app/modules/analytics.js]: Can we move this to a separate function like `getMixpanelConfig` ?

File: package.json [hunk 4]
```diff
@@ -83,11 +83,11 @@
     "react2angular": "4.0.2",
     "readable-stream": "2.3.3",
     "redux": "3.5.2",
-    "rendition": "4.27.3",
+    "rendition": "4.41.1",
```
Comment (zvin) [inline package.json]: Is this update intended?

Comment (thundron) [reply package.json]: Yes, it fixes an issue with an internal rendition package that bumped version and changed the folder structure which broke the builds

**Pull request merged**